src/src/Makefile.am

```diff
@@ -63,6 +63,7 @@ codeblocks_SOURCES = app.cpp \
 			startherepage.cpp \
 			switcherdlg.cpp \
 			threadsdlg.cpp \
+			uservardlgs.cpp \
```

Comment:
What about the XRC file moves? Are they correctly incorporate in the autotools build? Using make dist and then listing the generated archive is a good way to check.